mscore/preferences.cpp

```diff
@@ -1149,6 +1151,18 @@ void PreferenceDialog::resetShortcutClicked()
       active->setText(1, shortcut->keysToString());
       shortcutsChanged = true;
       }
+    
+void PreferenceDialog::saveShortcutListClicked()
+      {
+      QString saveFileName = QFileDialog::getSaveFileName(new QWidget(),"Save Shortcuts", tr("XML Files(*.xml)"));
```

Comment:
This is a mess, check again the documentation of getSaveFileName. Also see how files are saved all around the MuseScore code.